tests/_data/fixtures/models/InvoicesSchema.php

```diff
@@ -29,8 +27,6 @@ class InvoicesSchema extends Model
     public function initialize()
     {
         $this->setSource('co_invoices');
-        $this->setSchema(
-            getenv('DATA_MYSQL_NAME')
-        );
+        $this->setSchema($_ENV['DATA_MYSQL_NAME']);
```

Comment:
Should we use `env()` here?

Reply:
https://github.com/phalcon/cphalcon/pull/15834/commits/df5ff79a0e7d94c5ae798a2f59f4e400c6720264